app/controllers/api/discussions_controller.rb

```diff
@@ -2,22 +2,16 @@ class API::DiscussionsController < API::RestfulController
   load_and_authorize_resource only: [:show, :mark_as_read, :set_volume]
   load_resource only: [:create, :update]
 
-  def inbox_by_date
-    load_and_authorize_group if params[:group_id]
+  def discussions_for_dashboard
     @discussions = page_collection inbox_threads
     respond_with_discussions
   end
 
-  def inbox_by_organization
+  def discussions_for_inbox
```

Comment:
There's a bit of a fuzzy line. I'm trying to keep things on the angular
side called 'threads', while everything in existing loomio is a
'discussion'. Happy to look at moving the name transition line a little
(maybe no calling things 'threads' once we get to the controller) but fact
is we're always going to have a bit of an odd spot where we do something
like

```
def get_threads
  Discussion.find_stuff
end
```

On Mon, May 4, 2015 at 11:58 AM, Richard D. Bartlett <
notifications@github.com> wrote:

> In app/controllers/api/discussions_controller.rb
> <https://github.com/loomio/loomio/pull/2137#discussion_r29566164>:
>
> >      @discussions = page_collection inbox_threads
> >      respond_with_discussions
> >    end
> >
> > -  def inbox_by_organization
> > +  def discussions_for_inbox
>
> I am happy to be ignored as a pedant but: is there a reason we're mixing
> 'threads' and 'discussions' in these methods?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/loomio/loomio/pull/2137/files#r29566164>.
>